src/schema/v2/sellable.ts

```diff
@@ -39,5 +41,57 @@ export const Sellable = new GraphQLInterfaceType({
       type: GraphQLString,
       resolve: ({ sale_message }) => sale_message,
     },
-  },
+  }),
 })
+
+export const sharedSellableFields = {
+  // Note: while EditionSets require this field to perform mutation updates,
+  // since they're technically embedded documents on the artwork, they're not
+  // queryable in isolation, by ID. They exist only in the context of the artwork.
+  ...InternalIDFields,
+  availability: {
+    type: GraphQLString,
+  },
+  displayLabel: {
+    type: GraphQLString,
+    resolve: ({ title }) => title,
+  },
+  displayPriceRange: {
+    type: GraphQLBoolean,
+    resolve: ({ display_price_range }) => display_price_range,
+  },
+  internalDisplayPrice: {
+    type: GraphQLString,
+    resolve: ({ internal_display_price }) => internal_display_price,
+    description: "Price for internal partner display, requires partner access",
+  },
+  isInAuction: {
+    type: GraphQLBoolean,
+    description: "Is this artwork part of an auction?",
+    resolve: async ({ sale_ids }, _options, { salesLoader }) => {
```

Comment:
How would this work for EditionSets? Don't thing `sale_ids` are accessible from looking at EditionSet in isolation?

Reply:
yea - we should make sure all of these fields _can_ be correctly resolved for both an artwork and an edition set - otherwise just asking for bugs/confusion later.

it might be premature to even consolidate all of these fields (and resolvers) in `sharedSellableFields`. Maybe instead, artwork and edition set should _each_ define all of these fields within their types/files, along with the appropriate resolvers?

Reply:
Since this an MP type I *suspect* it's ok to make it easier to get the artwork properties from each edition set and define this as 'sellable' interface which can be either artwork or edition set. In fact we already  use it as such in some cases: https://github.com/artsy/gravity/blob/8aab8f403c395c03630d543dd564c407ca450552/app/models/domain/invoice_line_item.rb#L74 However for resolving I believe we need to be able to read the property from edition set object, no?  That is why we've added bunch of those `delegate` references: https://github.com/artsy/gravity/blob/8aab8f403c395c03630d543dd564c407ca450552/app/models/domain/edition_set.rb#L35

`sale_ids` are not accessible from EditionSet object directly as far as I can tell...

Reply:
Oh yes - that's what I'm getting at! Sharing the _resolvers_ here likely won't work b/c artwork + edition set, while they have a lot of overlap in JSON fields, aren't identical.

So we may still want to leave the field declarations here for documentation, but likely a bunch/all of these fields should exist in _both_ `Artwork` and `EditionSet` MP types.

And furthermore, b/c some of this needs to be resolved using artwork-level fields that aren't available on the edition set JSON (like `sale_ids`) - in addition to creating the correct resolvers on `Artwork` and `EditionSet`, we _also_ want to inject the `artwork` JSON into the edition set JSON for `EditionSet` resolvers to have access to.

Reply:
Resolved these, and others, to the artwork